gusto/timeloop.py

```diff
@@ -378,6 +377,89 @@ def timestep(self):
                 scheme.apply(self.x.np1(scheme.field_name), self.x.np1(scheme.field_name))
 
 
+class SplitPrescribedTransport(Timestepper):
+    """
+    Implements a timeloop where the physics terms are solved separately from
+    the dynamics, like with SplitPhysicsTimestepper, but here we define
+    a prescribed transporting velocity, as opposed to having the
+    velocity as a prognostic variable.
+    """
+
+    def __init__(self, equation, scheme, io, spatial_methods=None,
+                 physics_schemes=None,
+                 prescribed_transporting_velocity=None):
+        """
+        Args:
+            equation (:class:`PrognosticEquation`): the prognostic equation
+            scheme (:class:`TimeDiscretisation`): the scheme to use to timestep
+                the prognostic equation
+            io (:class:`IO`): the model's object for controlling input/output.
+            spatial_methods (iter,optional): a list of objects describing the
+                methods to use for discretising transport or diffusion terms
+                for each transported/diffused variable. Defaults to None,
+                in which case the terms follow the original discretisation in
+                the equation.
+            physics_schemes: (list, optional): a list of tuples of the form
+                (:class:`PhysicsParametrisation`, :class:`TimeDiscretisation`),
+                pairing physics parametrisations and timestepping schemes to use
+                for each. Timestepping schemes for physics can be explicit
+                or implicit. Defaults to None.
+            prescribed_transporting_velocity: (field, optional): A known
```

Comment:
Could you add a bit more description to this to explain that it can be a python function, taking time as an argument?

I see that the docstrings are wrong for the `PrescribedTransport` stepper, sorry about that!